docs/en/enterprise-edition/policy-reference/secrets-policies/secrets-policy-index/git-secrets-112.adoc

```diff
@@ -0,0 +1,49 @@
+== Azure Functions HTTP Trigger Key detected in code
+
+
+=== Policy Details
+
+[width=45%]
+[cols="1,1"]
+|===
+|Prisma Cloud Policy ID
+|TBD
+
+|Checkov Check ID
+|CKV_SECRET_112
+
+|Severity
+|LOW
+
+|Subtype
+|Build
+
+|Frameworks
+|Git
+
+|===
+
+
+=== Description
+
+An Azure Functions HTTP Trigger Key is essential for securing HTTP-triggered Azure Functions, allowing only authorized clients to invoke the function. This key ensures controlled access, which is crucial for preventing unauthorized data exposure and manipulation through the function. Secure management of this key is imperative to maintain the integrity and security of the applications reliant on Azure Functions.
+
+=== Fix - Buildtime
+
+*Azure*
+
+If an Azure Functions HTTP Trigger Key is inadvertently exposed, immediate actions should be taken to revoke the old key and generate a new one, thus securing access to the function:
```

Comment:
If your Azure Functions HTTP Trigger Key is exposed, it's crucial to take immediate action to revoke the old key and generate a new one in order to secure access to the function. By replacing the exposed key and updating all applications  accordingly, you effectively safeguard your Azure Function from unauthorized access.